sdks/python/apache_beam/io/gcp/bigquery_file_loads.py

```diff
@@ -116,6 +115,11 @@ def _make_new_file_writer(file_prefix, destination):
   return file_path, fs.FileSystems.create(file_path, 'application/text')
 
 
+def _file_size(file_path):
```

Comment:
Try different approaches

Reply:
1. Calculate size when writing each row
2. return filesystem object in _make_new_file_writer so .size() is accessible